mappings/net/minecraft/client/render/debug/HiveFarmDebugRenderer.mapping

```diff
@@ -0,0 +1,97 @@
+CLASS net/minecraft/class_4703 net/minecraft/client/render/debug/HiveFarmDebugRenderer
```

Comment:
Why "hive farm"?

Reply:
It deals with hives/nests and bees, and fits with the theming of the `VillageDebugRenderer` which deals with villagers and profession workstations.

Also the code looked really similar to the `VillageDebugRenderer` when I first saw it.

I think it's an apt description.

Reply:
how about BeeDebugRenderer as both hive and flower info are around bees

Reply:
If we do, I'd like to make the other one `VillagerDebugRenderer`.

Reply:
yes please. we have villager type instead of village type in registries!

Reply:
:O